studio/components/layouts/AccountLayout/AccountLayout.tsx

```diff
@@ -99,7 +99,7 @@ const AccountLayout: FC<Props> = ({ children, title, breadcrumbs }) => {
           key: 'ext-guides',
           icon: `${router.basePath}/img/book.svg`,
           label: 'Guides',
-          href: 'https://supabase.com/docs',
+          href: 'https://supabase.com/docs/guides/platform',
```

Comment:
is this correct? You want to send to the platform docs, not the docs homepage?

Reply:
ok, on second thought, it does make more sense to show the more general docs homepage as first link

maybe I acted a bit too quickly when I thought, why are guides not linking to the guides page!